client/src/pages/private/ParticipantTable.js

```diff
@@ -136,9 +136,11 @@ export default () => {
   const { auth } = AuthContext.useAuth();
   const roles = useMemo(() => auth.user?.roles || [], [auth.user?.roles]);
   const sites = useMemo(() => auth.user?.sites || [], [auth.user?.sites]);
+  let hasWithdrawnParticipant = false;
```

Comment:
This seems to be only used in `filterData`, if that's right can it be initialized in that function?

Reply:
Also used in `runAsync ()`